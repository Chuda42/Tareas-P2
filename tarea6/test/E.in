asociarEnMap 1 15.0
asociarEnMap 10 25.0
existeAsociacionEnMap 5
valorEnMap 1
estaLlenoMap
estaVaciaCP
estaEnCP 1
rangoCP
hacerVecinos 2 1 5.0
hacerVecinos 3 1 5.0
hacerVecinos 10 1 5.0
hacerVecinos 9 1 5.0
hacerVecinos 8 1 5.0
hayMParejas
sonVecinos 1 2
sonVecinos 6 1
distancia 1 2
vecinos 2
vecinos 9
vecinos 6
cantidadVertices
hacerVecinos 2 3 1.0
hacerVecinos 2 10 1.0
hacerVecinos 2 9 1.0
hacerVecinos 2 8 1.0
hacerVecinos 4 7 2.0
hacerVecinos 7 5 2.0
hacerVecinos 4 5 2.0
hacerVecinos 10 9 2.0
hacerVecinos 10 8 2.0
hacerVecinos 10 3 2.0
hacerVecinos 8 3 2.0
hacerVecinos 9 3 2.0
hacerVecinos 9 8 2.0
hacerVecinos 6 1 0.1
hacerVecinos 6 2 0.1
hacerVecinos 6 3 0.1
hacerVecinos 6 8 0.1
hacerVecinos 6 9 0.1
hacerVecinos 6 10 0.1
hacerVecinos 4 1 0.5
hacerVecinos 4 2 0.5
hacerVecinos 4 3 0.5
hacerVecinos 4 9 0.5
hacerVecinos 4 10 0.5
hayMParejas
hacerVecinos 4 8 0.5
hayMParejas
Fin

