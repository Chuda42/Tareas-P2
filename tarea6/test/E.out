1>Establecida la asociación.
2>Establecida la asociación.
3>No existe asociacion.
4>15.00
5>No lleno.
6>Vacia.
7>No está.
8>10.
9>Nuevos vecinos: 2 1 5.00.
10>Nuevos vecinos: 3 1 5.00.
11>Nuevos vecinos: 10 1 5.00.
12>Nuevos vecinos: 9 1 5.00.
13>Nuevos vecinos: 8 1 5.00.
14>NO hay M parejas.
15>Son vecinos.
16>NO son vecinos.
17>5.00
18>1 
19>1 
20>
21>10.
22>Nuevos vecinos: 2 3 1.00.
23>Nuevos vecinos: 2 10 1.00.
24>Nuevos vecinos: 2 9 1.00.
25>Nuevos vecinos: 2 8 1.00.
26>Nuevos vecinos: 4 7 2.00.
27>Nuevos vecinos: 7 5 2.00.
28>Nuevos vecinos: 4 5 2.00.
29>Nuevos vecinos: 10 9 2.00.
30>Nuevos vecinos: 10 8 2.00.
31>Nuevos vecinos: 10 3 2.00.
32>Nuevos vecinos: 8 3 2.00.
33>Nuevos vecinos: 9 3 2.00.
34>Nuevos vecinos: 9 8 2.00.
35>Nuevos vecinos: 6 1 0.10.
36>Nuevos vecinos: 6 2 0.10.
37>Nuevos vecinos: 6 3 0.10.
38>Nuevos vecinos: 6 8 0.10.
39>Nuevos vecinos: 6 9 0.10.
40>Nuevos vecinos: 6 10 0.10.
41>Nuevos vecinos: 4 1 0.50.
42>Nuevos vecinos: 4 2 0.50.
43>Nuevos vecinos: 4 3 0.50.
44>Nuevos vecinos: 4 9 0.50.
45>Nuevos vecinos: 4 10 0.50.
46>NO hay M parejas.
47>Nuevos vecinos: 4 8 0.50.
48>Hay M parejas.
49>Fin.
